### PR TITLE
First Commit for Mobile Security Framework, Created all actions and d…

### PR DESCRIPTION
**Pull request by cs-adarsh-gupta**
#### UTCs: 
 
- [x] Connector installation verified. 
- [x] Connector logo verified. 
- [x] Actions and Playbook list verified. 
- [x] Playbook tags verified. 
- [x] All Playbook are in info mode verified. 
- [x] All Playbook are in inactive mode verified.

### CODE REVIEW

File: mobile-security-framework/connector.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+    def check_health(self, config=None):
+        try:
+            config['connector_info'] = {"connector_name": self._info_json.get('name'),
+                "connector_version": self._info_json.get('version')}
+            return _check_health(config)
+        except Exception as err:
+            raise ConnectorError(err)
+
+    def on_app_start(self, config, active):
```
Comment (cs-omkar-kumbhar) [inline mobile-security-framework/connector.py]: these unused functions can be removed

File: mobile-security-framework/info.json [hunk 2]
```diff
@@ -0,0 +1,701 @@
...
+          "type": "checkbox",
+          "editable": true,
+          "visible": true,
+          "required": false,
+          "tooltip": "Tick if you want to Re scan the File, Default is False.",
+          "description": "Tick if you want to Re scan the File, Default is False.",
+          "value": false
+        }
+      ],
+      "output_schema": {}
```
Comment (cs-omkar-kumbhar) [inline mobile-security-framework/info.json:151]: This action does not return any output ?

File: mobile-security-framework/info.json [hunk 3]
```diff
@@ -0,0 +1,701 @@
...
+          "title": "Hash",
+          "type": "text",
+          "editable": true,
+          "visible": true,
+          "required": true,
+          "tooltip": "Specify the Hash of the Scan File.",
+          "description": "Specify the Hash of the Scan File."
+        }
+      ],
+      "output_schema": {}
```
Comment (cs-omkar-kumbhar) [inline mobile-security-framework/info.json:431]: output schema missing ?

File: mobile-security-framework/info.json [hunk 4]
```diff
@@ -0,0 +1,701 @@
...
+          "title": "Hash of Second Scan",
+          "type": "text",
+          "editable": true,
+          "visible": true,
+          "required": true,
+          "tooltip": "Specify the Hash of the Second Scan File.",
+          "description": "Specify the Hash of the Second Scan File."
+        }
+      ],
+      "output_schema": {}
```
Comment (cs-omkar-kumbhar) [inline mobile-security-framework/info.json:514]: output schema missing?

File: mobile-security-framework/info.json (hunk 2 above)
Comment (cs-adarsh-gupta) [reply mobile-security-framework/info.json:151]: The outputs of the this actions are different for every file and are very large in size, So after discussion it's decided to leave it as blank.

File: mobile-security-framework/info.json (hunk 3 above)
Comment (cs-adarsh-gupta) [reply mobile-security-framework/info.json:431]: The outputs of the this actions are different for every file and are very large in size, So after discussion it's decided to leave it as blank

File: mobile-security-framework/info.json (hunk 4 above)
Comment (cs-adarsh-gupta) [reply mobile-security-framework/info.json:514]: The outputs of the this actions are different for every file and are very large in size, So after discussion it's decided to leave it as blank

**Review by cs-omkar-kumbhar — APPROVED** [review]

**Pull request merged**